java/lib/src/main/java/io/amplica/graphsdk/models/Actions.java

```diff
@@ -217,7 +217,7 @@ protected java.lang.Object newInstance(
 
```

Comment:
Shouldn't all of these generated files be deleted from the repo now that they're being generated in the build process?